scripts/soroban_repo_to_horizon_repo.sh

```diff
@@ -16,9 +16,9 @@ fi
 
```

Comment:
I'm inclined to say we should remove this script instead of keep it. In all cases where we specify a version of rpc we also specify a version explicitly of horizon. This script is only used by the makefile, but it should be easier now-days to pick versions of horizon that match versions of soroban-rpc. Wdyt?

Reply:
I dont have enough context on script removal part but this PR is pointing to old version of rpc, latest released soroban-rpc version is `20.3.2`  so we need to update that.  overall yes, we should definitely update quickstart so that it points to new soroban-rpc. Are there any edge cases here? cc @mollykarcher  (if not I can refactor this PR and land)

Reply:
I think this can follow your suggestion and be landed with a little more adjustment like you mentioned for 20.3.2, regarding `build-future.yml`, that refers to futurenet, and I don't think that network has been updated to core 20.2.0 , which would mean leave it's soroban-tools/tag as-is, unless the same older tag is copied on soroban-rpc repo, the futurenet network option probably needs to be deprecated from quickstart on a separate ticket.

can sanity test pr locally by running the make targets for `latest` or `testing` image variants, and run the image and watch console output to confirm it joins the network:
```
docker run --rm -it     -p "8000:8000"   --name stellar     stellar/quickstart:latest|testing --testnet
``` 

can use `--local` for network also on either of those image variants, quickstart will run a standalone network in the container using CORE_REF version, rather than connecting to remote.